Files/Scripts/Options/The Sealed Palace.psm1

```diff
@@ -136,6 +164,9 @@ function ByteOptions() {
     if (IsDefault $Redux.Hitbox.KokiriSword       -Not)   { ChangeBytes -Offset "B9D288"  -Values (ConvertFloatToHex $Redux.Hitbox.KokiriSword.Value) }
     if (IsDefault $Redux.Hitbox.MasterSword       -Not)   { ChangeBytes -Offset "B9D284"  -Values (ConvertFloatToHex $Redux.Hitbox.MasterSword.Value) }
     if (IsDefault $Redux.Hitbox.GiantsKnife       -Not)   { ChangeBytes -Offset "B9D28C"  -Values (ConvertFloatToHex $Redux.Hitbox.GiantsKnife.Value) }
+    if (IsDefault $Redux.Hitbox.BrokenKnife	      -Not)   { ChangeBytes -Offset "BB5AEC"  -Values (ConvertFloatToHex $Redux.Hitbox.BrokenKnife.Value) }
```

Comment:
Limit your changes to just the DekuStick

Reply:
The broken knife and megaton hammer were missing, so I added those for 'sealed palace'.